upgrade_deadlines/repair.py

```diff
@@ -25,20 +25,50 @@
 LMS_PASSWORD = os.environ.get('LMS_PASSWORD')
 
 
+def get_access_token():
+    response = requests.post(
+        f'{OAUTH_ACCESS_TOKEN_URL}/access_token',
```

Comment:
Same here, does OAUTH_ACCESS_TOKEN_URL not end in `/`? Looking at this code, seems like some URLs end with the forward slash and some don't

Reply:
Same for OAUTH_ACCESS_TOKEN_URL: 
https://github.com/openedx/ecommerce-scripts/pull/186/files#diff-9f620f4b04a54f97b94943e5882d47c7e41892f3e1c791571dd2d1b055e0194cR17